zha/application/platforms/sensor/const.py

```diff
@@ -390,3 +390,36 @@ class SensorDeviceClass(enum.StrEnum):
     SensorDeviceClass.ENUM,
     SensorDeviceClass.TIMESTAMP,
 }
+
+
+class AnalogInputStateClass(enum.Enum):
```

Comment:
The ZCL specifies units for each value of ApplicationType, currently we are using units as specified in EngineeringUnits. Both attributes are optional, do we leave this as is, where sitting a ApplicationType only sets the device class, should it also set the units, if so, what should happen if both attributes are set?